backend/src/routes/metrics.ts

```diff
@@ -0,0 +1,48 @@
+/* Copyright Contributors to the Open Cluster Management project */
+
+import { Http2ServerRequest, Http2ServerResponse } from 'http2'
+import Prometheus, { register } from 'prom-client'
+import { logger } from '../lib/logger'
+
+interface PageData {
+  page: string
+}
+
+// register.setDefaultLabels({
+//   // label to identify customer?
+// })
```

Comment:
Data should be anonymous. We'll know the cluster reporting this metric and can use it to find the customer account.